Overview topics for styling, components, patterns (#372)
IgniteUI/indigo-design-docfx: en/style/styling-overview.md

---
title: Styling Overview - Design System Styling
_description: The Indigo.Design System contains everything for you to be able to define themes for both Components and Patterns.
_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
---

# Styling Overview

> The **Indigo.Design System** provides elaborate means for styling all 60+ UI components that also map to Ignite UI for Angular controls. The theming engine of Infragistics' Angular product, through powerful sass functions and mixins, allows for the creation of custom color themes that can restyle the whole product in just a few lines of code in a similar fashion. 

Reviewer: ```suggestion
The **Indigo.Design System** provides elaborate means for styling all 60+ UI components that also map to Ignite UI for Angular controls. The theming engine in our Angular product allows for the creation of custom color themes that can restyle the whole product in just a few lines of code in a similar fashion.
```

The Sketch library has a few pages prefixed with a 🎨 emoji that provide assets and a powerful styling infrastructure upon which the rest of the library is built:

- 🎨 Material Icons
- 🎨 Colors
- 🎨 Typography
- 🎨 Elevation
- 🎨 Illustrations

> In Sketch the colors and elevations exist as `Layer Styles`, while the typographies are defined as `Text Styles`, which allows to quickly change the theme for any given design we have created with the **Indigo.Design System**.

Reviewer: ```suggestion
In Sketch, the colors and elevations exist as `Layer Styles`, while typography is defined as `Text Styles`, which allows you to quickly change the theme for any given design we have created with the **Indigo.Design System**.
```

<img class="responsive-img" src="../images/theme_overview_default.png" />

<img class="responsive-img" src="../images/theme_overview_dark.png" />

<img class="responsive-img" src="../images/theme_overview_vibrant.png" />

To find more:

> - [Material Icons](material-icons.md) - a collection of 150+ Material Icons extended with 250+ more icons created by us according to the Material icon design guidelines, which you can easily extend even furhter

Reviewer 2: furhter -> further
